api/nakadi-event-bus-api.yaml

```diff
@@ -1245,6 +1245,115 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /timelines:
```

Comment:
I thought it will be better to have timelines under event type - like `/event-types/{event-type}/timelines`